### ISSUE TITLE
Give User way to "SHARE" email with a colleague, where the colleague can tweak text and then say "OK I'm ready"

### ISSUE THREAD

**Issue by daver78** [coa0329/BhaangWeb#166]
Given the insight-provoking experience watching @Mcfarsheed and @paagla123 converse for ~2 hours to get an email right, it gave rise to the idea that perhaps the following is useful:

1- Someone like @paagla123 launches a campaign, say, Promote a Blog Post.

2- He configures everything based on assets provided to him by the marketer (or his boss, his client, etc.)

3- At some point in the process, @paagla123 sends some tests to the marketer (or his boss, client, etc.).  That person/people have numerous edits they want to make.

4- @paagla123 can go through a lengthy and painful process of making slight tweaks until the other person is happy (especially hard for people who are mismatched in their level of attention to detail) ... or:

**5- @paagla123 can press a "Share" button somewhere in the email UI, and enter in @Mcfarsheed's address (or his boss, client, etc.).  

6- The other person receives a link where they can edit the email's content, send themselves a test, preview it, etc.  Then when they're happy, they can press a "send back to @paagla123" button.  @paagla123 is notified that so-and-so is done and is ready for the email to launch.**

**Comment by daver78** [comment]
@paagla123 and @Mcfarsheed What do you think?

@javpowell What do you think of this based on your personal experiences?

**Comment by Mcfarsheed** [comment]
Fantastic!

**Comment by Mcfarsheed** [comment]
I think it's great!

**Comment by daver78** [comment]
@Mcfarsheed Part of me wonders why the "you" in this scenario doesn't just log into DV and do it yourself.  But my answer to myself is that if you're @paagla123's boss, or @paagla123's client, @paagla123 likely doesn't want to expose you to everything else in DV.  

Am I making sense?

Yes, DV is easy.  But there are still reasons why @paagla123 would only want to bring you into the workflow/approval process vis-a-vis the email in specific.

Yes, no?

**Comment by javpowell** [comment]
Hmmmmm, its hard to say how this would be in the real world, sounds like a
great idea but I wonder if people would use it. Kinda like sharing a report
view, I feel like its a cool option but people usually just add users to
the dashboard instead to get full control of reporting.


*Jason Voigt*
*Client Services Manager*
jvoigt@mindfireinc.com | *949.579.9840*
http://www.mindfirestudio.com/

**Comment by daver78** [comment]
@paagla123 I think there are analogs to this in the outside world -- things like sharing a Google Doc for a round of edits/feedback, etc.  

Also, I believe that when there is a mismatch in 'attention to detail' (say one person is very attentive to details, and the other person can't see those things), then if the person with the detailed eyes is the "Customer", they'll benefit from being able to tweak to their heart's content.  

This also makes it easy for the other person (the one building the campaign) to off-load that stress to the person who actually 'enjoys' getting it perfect.

We'll hold this in the backlog and see what we do about it.

**Comment by Mcfarsheed** [comment]
Well if it's an easy enough tool, the "me" in this case would assign the
task of weekly deploying campaigns to a marketing coordinator or something
and in that case I would want to see tests and approve every single one
then provide her/him feedback to go update.

Sincerely,

Mackenzi
MindFireInc.
949-842-4388
www.mindfireinc.com

**Comment by daver78** [comment]
@Mcfarsheed Are you saying that you would prefer the workflow you had yesterday with Kush?  That's what I read from "_I would want to see tests and approve every single one then **provide her/him feedback to go update**_."

**Comment by Mcfarsheed** [comment]
You mean prefer that to going in and doing it myself? Well I think if I
ideally had a marketing coordinator who was in the marketing role, this
should be something that they take on. Here's how the workflow would go:
1. I assign topic/assignment (well more like discuss or it's all mapped out
weeks in advance)
2. Marketing coordinator writes email copy, creates image, creates TA,
creates links with UTM's, etc..
3. They send me the above for approval and I provide edits etc.. until I
think it looks good
3. They get everything ready and send me tests
4. I reply with edits
5. They send again

repeat 4 and 5 until perfect - hopefully, as they learn and continue to do
this, the edits become less and less in quantity.

Over time the goal is 95% autonomy with a quick approval.

Make sense?



Sincerely,

Mackenzi
MindFireInc.
949-842-4388
www.mindfireinc.com

**Comment by daver78** [comment]
@Mcfarsheed Hmmm one of us is confused.  We'll talk phone later :)

**Comment by Mcfarsheed** [comment]
Ok you can call me whenever. I don't know what you are referring to but the
process I outlined above is exactly how I see it going down in an ideal
situation.

Me - Manager
Marketing Coordinator - Executer

The exact steps are listed.

Sincerely,

Mackenzi
MindFireInc.
949-842-4388
www.mindfireinc.com

**Comment by paagla123** [comment]
@Mcfarsheed it was very confusing yesterday. I can see why you had the problem with Kathy way back. You cannot keep saying over Skype what you want. You have to take time to write it down, which is what you did eventually in GDoc. Might as well have done that directly in DV. Same thing - right?

VROOM! VROOM!!

**Comment by Mcfarsheed** [comment]
Exactly! But there is one main difference - you and kathy are not the
marketing person so there's a disconnect and confusion.

So, in this case, I would be the person doing the campaign. BUT like I said
in my previous email, I would want a marketing coordinator to do it. The
main difference is that this person would ALSO be writing the copy etc.. so
it wouldn't be me telling them and then them executing, they would be
driving the entire thing after we brainstormed which would save the
confusion.

As it stands right now I have Ashleigh writing copy and creating the image
and then I edit the copy and make changes before doing testing with you.
There's no confusion with her executing that and me testing. The marketing
coordinator would do exactly what she does PLUS set up emails (which I
typically do) and I would want to just be sent the final test for approval
+ she would execute any feedback I have on that.

I think a big disconnect with these scenarios is you are not the marketing
person so you're relying on me to give you content and changes etc..



Sincerely,

Mackenzi
MindFireInc.
949-842-4388
www.mindfireinc.com

**Comment by paagla123** [comment]
@Mcfarsheed vroom vroom!
